AFQ/tasks/data.py

```diff
@@ -449,6 +451,103 @@ def gq_ai(gq_params):
     return AI, dict(GQParamsFile=gq_params)
 
 
+@pimms.calc("rumba_model")
+def rumba_model(gtab,
+                rumba_wm_response=[0.0017, 0.0002, 0.0002],
+                rumba_gm_response=0.0008,
+                rumba_csf_response=0.003,
+                rumba_n_iter=600):
+    """
+    fit for RUMBA-SD model as documented on dipy reconstruction options
+
+    Parameters
+    ----------
+    rumba_wm_response: 1D or 2D ndarray or AxSymShResponse.
+        Able to take response[0] from auto_response_ssst.
+        default: array([0.0017, 0.0002, 0.0002])
+    rumba_gm_response: float, optional
+        Mean diffusivity for GM compartment.
+        If None, then grey matter volume fraction is not computed.
+        Default: 0.8e-3
+    rumba_csf_response: float, optional
+        Mean diffusivity for CSF compartment.
+        If None, then CSF volume fraction is not computed.
+        Default: 3.0e-3
+    rumba_n_iter: int, optional
+        Number of iterations for fODF estimation.
+        Must be a positive int.
+        Default: 600
+    """
+    return RumbaSDModel(
+        gtab,
+        wm_response=np.asarray(rumba_wm_response),
+        gm_response=rumba_gm_response,
+        csf_response=rumba_csf_response,
+        n_iter=rumba_n_iter,
+        recon_type='smf',
+        n_coils=1,
+        R=1,
+        voxelwise=False,
+        use_tv=False,
+        sphere=default_sphere,
+        verbose=True)
+
+
+@pimms.calc("rumba_params")
+@as_file(suffix='_odfmodel-RUMBA_desc-diffmodel_dwi.nii.gz')
+@as_img
+def rumba_params(rumba_model, data, brain_mask):
+    """RUMBA SHM Parameters"""
```

Comment:
```suggestion
    """Fits the RUMBA SD model. """
```

Would be good to finish this docstring, including a "Parameters" and a "Returns" sections.

Reply:
@36000 would this docstring work? I don’t think there’s much room for user configuration in this function, and I’m not too sure what `meta` does… took a stab at defining what the function returns by looking at `QBallTP.py` 
  
   ```
"""
   Takes the fitted RUMBA-SD model as input and returns the spherical harmonics coefficients (SHM).

    Parameters
    ----------
    rumba_fit: use output from rumba_fit as input for rumba_params

    Returns
    ----------
    rumba_shm: spherical harmonics coefficients from an ODF without the isotropic and anisotropic diffusion components

    """
```

Reply:
This is a part of the task system so we typically only have parameters when theres user input via named arguments, as you point out Howard. We also typically don't have returns because we usually just describe it in the main docstring. These task functions are all only called by pimms. So I think we could do something like this is good:
```
Takes the fitted RUMBA-SD model as input and returns the spherical harmonics coefficients (SHM).
```